src/pages/index.js

```diff
@@ -1,94 +1,217 @@
-import "./index.css";
+import Api from "../components/Api.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
-
+import "./index.css";
 import {
-  initialCards,
-  cardsWrap,
-  profileEditModal,
-  profileEditForm,
   profileEditOpenButton,
-  profileEditCloseButton,
-  addCardModal,
-  addCardForm,
   addCardOpenButton,
-  addCardCloseButton,
-  imageModal,
-  imageModalPreview,
-  previewModalCloseButton,
-  modalCaption,
-  profileTitle,
-  profileDescription,
+  editAvatarOpenButton,
   profileTitleInput,
   profileDescriptionInput,
-  newCardTitleInput,
-  newCardLinkInput,
   config,
-  cardData,
 } from "../utils/constants.js";
 
-const newUserInfo = new UserInfo(".profile__title", ".profile__description");
+/* ------------------------- CREATE AND RENDER CARDS ------------------------ */
+function createCard(cardData) {
+  const cardElement = new Card(
+    cardData,
+    "#card-template",
+    handleLikeClick,
+    handleDeleteClick,
+    (link, name) => {
+      imagePreview.open(link, name);
+    }
+  );
+  return cardElement.getView();
+}
 
-const addCardPopup = new PopupWithForm(
-  "#profile-add-card-modal",
-  handleAddCardSubmit
-);
-addCardPopup.setEventListeners();
+function renderCard(cardData) {
+  const element = createCard(cardData);
+  cardSection.addItem(element);
+}
 
-const editProfilePopup = new PopupWithForm(
-  "#profile-edit-modal",
-  handleProfileEditSubmit
-);
-editProfilePopup.setEventListeners();
+/* ------------------------ USER INFO AND INITAL CARS ----------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "60774818-2b72-4a3b-8d83-5b2abf9c8a5e",
+    "Content-Type": "application/json",
+  },
+});
 
-const imagePreview = new PopupWithImage("#popup-modal");
-imagePreview.setEventListeners();
+api
+  .loadPageContent()
+  .then(([cards, userData]) => {
+    newUserInfo.setUserAvatar(userData.avatar);
+    newUserInfo.setUserInfo({
+      name: userData["name"],
+      about: userData["about"],
+      id: userData["_id"],
```

Comment:
`   id: userData["_id"],` is not needed here because you don't use it inside `setUserInfo`

So, you can simplify it here

```js

    newUserInfo.setUserInfo(userData);
```